### DISCUSSION TITLE
Regolith does not respond to keyboard input

### DISCUSSION THREAD

**Discussion by Vasja1986** [regolith-linux/regolith-desktop#1002] [Q&A] [answered]
I just acquired a laptop and put a fresh Debian 12 install without a desktop environment on it. Then I proceeded to install regolith (the wayland session) and lightdm on it. So far so good. But I immediately ran into problems. i3status is for whatever reason not marked as a dependency of regolith, so I had to manual install it (luckily a web search with the error message rendered a useful result). The bigger problem is that my desktop is not responding to keyboard input. Upon first launch of a regolith session I get a screen asking whether I want to modify default keybindings. I then press Escape to continue with the default keybindings. After that no single keybinding does work. I only can click on the network icon in the bottom right of the screen and pick a network. I can enter the password for my network, so keyboard is definitely working.

I am puzzled and am not sure how to debug this issue. Any advice is appreciated.

**Comment by Vasja1986** [comment, marked as answer]
Turns out, that lightdm started an x11 session instead of a wayland session. Under the X11 session I was able to spawn a terminal using the ALT+ENTER key-combination (accidentally discovered this by a stroke of luck). To remedy the situation I removed /usr/share/xsession/lightdm.desktop. After a reboot Regolith is launched as a wayland session upon login and the keybindings all seem to work.

**Reply by kgilmer** [reply]
Glad you figured out the problem.  FWIW lighten should let you select which session to start in the login screen
